grub-core/loader/i386/linux.c

```diff
@@ -826,7 +826,7 @@ grub_cmd_linux (grub_command_t cmd __attribute__ ((unused)),
 
   len = sizeof (linux_params) - sizeof (lh);
 
-  grub_memcpy (&linux_params + sizeof (lh), kernel + kernel_offset, len);
+  grub_memcpy ((char *)&linux_params + sizeof (lh), kernel + kernel_offset, len);
```

Comment:
Being picky, the coding style puts a space after casts, but this is unlikely to be upstreamed since a newer implementation is being proposed, so not a big deal.

Reply:
Ack, I re-pushed to fix the spacing.